code/game/machinery/computer/arcade.dm

```diff
@@ -1112,6 +1115,36 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 	explosion(loc, 2,4,8, flame_range = 16)
 	qdel(src)
 
+// ** AMPUTATION ** //
+
+/obj/machinery/computer/arcade/amputation
+	name = "Mediborg's Amputation Adventure"
+	desc = "A picture of a blood-soaked medical cyborg flashes on the screen. The mediborg has a speech bubble that says, \"Put your hand in the machine if you aren't a <b>coward!</b>\""
+	icon_state = "arcade"
+	circuit = /obj/item/circuitboard/computer/arcade/amputation
+
+/obj/machinery/computer/arcade/amputation/attack_hand(mob/user)
+	if(!iscarbon(user))
+		return
+	var/mob/living/carbon/c_user = user
+	var/obj/item/bodypart/l_arm = c_user.get_bodypart(BODY_ZONE_L_ARM)
+	var/obj/item/bodypart/r_arm = c_user.get_bodypart(BODY_ZONE_R_ARM)
+	to_chat(c_user, "<span class='warning'>You move your hand towards the machine, and begin to hesitate as a bloodied guillotine emerges from inside of it...</span>")
+	if(do_after(c_user, 50, target = src))
+		to_chat(c_user, "<span class='userdanger'>The guillotine drops on your arm, and the machine sucks it in!</span>")
+		playsound(loc, 'sound/weapons/slice.ogg', 25, 1, -1)
+		if(l_arm)
+			l_arm.dismember()
+			qdel(l_arm)
+		else
+			r_arm.dismember()
```

Comment:
This will cause a runtime if the user lacks a right arm

Reply:
Good point. Would prefer to fix it by making this whole thing just target the active arm, but again, there doesn't seem to be a way to do that...